lms/static/js/student_account/views/account_settings_factory.js

```diff
@@ -221,14 +221,14 @@
             aboutSectionsData.push(socialFields);
             // Add the mobytize_id and mobytizetoken fields
             mobytizeFields = {
```

Comment:
I guess it would be more consistent to rename this variable and modify the comment above as well.

```js
// Add the profitonomy_id and profitonomy_token fields
profitonomyFields = {
```

Reply:
I tried not to change variables, but only visual appearance. Logic in both platforms (Skillonomy and Profitonomy) works the same way, so I thought it will be better to leave variables as they are.